### PR TITLE
docs: add route_configuration_options.md

### PR DESCRIPTION
**Pull request by DammneD**

### CODE REVIEW

File: docs/routing/route_configuration_options.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Route configuration options
```
Comment (nightnei) [inline docs/routing/route_configuration_options.md:1]: now our `docs/assets` directory looks like a dump) let's create subdirectory `routes` and relocate all related assets

File: docs/routing/route_configuration_options.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+ ## Route domains
+ ![ILC registry domains field](../assets/domain-field.png)
```
Comment (nightnei) [inline docs/routing/route_configuration_options.md]: before screenshot add some introduction information
for example - why developers should specify some domain name. in which cases it's necessary and when it's useless. just imagine, that you are explaining someone - what is it and which value has "router domain"

File: docs/routing/route_configuration_options.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+ - You can add new domain in **Router domains** menu.
+
+ - Domain name, must be without protocol, etc.
```
Comment (nightnei) [inline docs/routing/route_configuration_options.md:8]: remove it - it's should be written in documentation about "Router domains" page

File: docs/routing/route_configuration_options.md (hunk 2 above)
Comment (nightnei) [reply docs/routing/route_configuration_options.md]: Unlike previous documentation, this has independent parts. So it should be not hard to create own description (introductional information) for each of them. So do it please for every part of the current doc.

**Review by nightnei — CHANGES_REQUESTED** [review]
Unlike previous documentation, this has independent parts. So it should be not hard to create own description (introductional information) for each of them. So do it (https://github.com/namecheap/ilc/pull/349#discussion_r784903476) please for every part of the current doc.

**Comment by StyleT** [comment]
@nightnei @DammneD guys, let's make new pages visible here https://microfrontends.online/

See:
- https://microfrontends.online/CONTRIBUTING/#documentation-website
- https://github.com/namecheap/ilc/blob/master/mkdocs.yml#L21-L56

**Review by nightnei — APPROVED** [review]

**Pull request merged**